piq/brisque.py

```diff
@@ -48,11 +48,10 @@ def brisque(x: torch.Tensor,
                       f'More info is available at https://github.com/photosynthesis-team/piq/pull/79 and'
                       f'https://github.com/pytorch/pytorch/issues/38869.')
 
-    _validate_input(input_tensors=x, allow_5d=False, kernel_size=kernel_size)
+    _validate_input(input_tensors=x, allow_5d=False, kernel_size=kernel_size, data_range=data_range)
     x = _adjust_dimensions(input_tensors=x)
 
-    assert data_range >= x.max(), f'Expected data range greater or equal maximum value, got {data_range} and {x.max()}.'
-    x = x * 255. / data_range
+    x = x / float(data_range) * 255
```

Comment:
What is the purpose of the following type conversion: `float(data_range)`? `data_range` (according to the type annotation) is either `int` of `float`. `\` operator always produces `float` result, hence the explicit type conversion is not needed. 
```suggestion
    x = x / data_range * 255
```

Please change this pattern (`float(data_range)`) in all other places if agree.